lib/components/map/route-preview-overlay.tsx

```diff
@@ -15,24 +15,31 @@ import {
 type Props = {
   from: Location
   geometries: string[]
+  mainPanelContent: number | null
   to: Location
   visible?: boolean
 }
 /**
  * This overlay will display thin gray lines for a set of geometries. It's to be used
  * as a stopgap until we make full use of Transitive!
  */
-const RoutePreviewOverlay = ({ from, geometries, to, visible }: Props) => {
+const RoutePreviewOverlay = ({
+  from,
+  geometries,
+  mainPanelContent,
+  to,
+  visible
+}: Props) => {
   // Center the map over the endpoints when this overlay is shown.
   const { current: map } = useMap()
   useEffect(() => {
-    if (visible) {
+    if (visible && mainPanelContent === null) {
```

Comment:
What does adding this `mainPanelContent` check do?

Reply:
Allows us to move the map in the route viewer and nearby view!